README.rst

```diff
@@ -88,7 +88,8 @@ Documentation
 For App Users
 .............
 
-App documentation doesn't yet exist. Watch this space.
+Documentation of the PyMedPhys application's graphical user interface doesn't
+yet exist. Watch this space.
```

Comment:
While I was in the area I saw that this read like "no documentation at all" existed. Clarified that it meant documentation of the GUI.